src/web_node.ml

```diff
@@ -66,7 +68,7 @@ let removeChild n child = n##removeChild child
 
 let insertBefore n child refNode = n##insertBefore child refNode
 
-let remove n child = n##remove child
```

Comment:
We have `n##remove : unit -> unit` from type `t`. If `remove` is meant to be an impure function which removes a node from the DOM, it's type should be `val remove: t -> unit`. A correct implemention would be `let remove n = n##remove ()`.

Reply:
Ahh that was entirely a copy-paste munge!  Good catch!  ^.^

Hmm, I'm not even using this function it looks like, maybe it should not even exist...